cms/static/sass/views/_group-configuration.scss

```diff
@@ -59,12 +59,12 @@
           @extend %cont-truncated;
           @extend %t-title5;
           @extend %t-strong;
-          margin-right: ($baseline*14);
+          @include margin-right($baseline*14);
           color: $black;
 
           .toggle {
+            @include padding-right($baseline);
```

Comment:
Well, it has been a very long time since I did this. As far as I remember, I flipped two things (the positive to negative above) and this is the other one to make the page look better in LTR as well.

It was based on my judgment of how it would be consistent with the other parts of the studio.